health/egov-persister/plan-service-persister.yml

```diff
@@ -198,6 +198,75 @@ serviceMaps:
             - jsonPath: $.PlanConfiguration.auditDetails.lastModifiedBy
             - jsonPath: $.PlanConfiguration.auditDetails.lastModifiedTime
 
+    - version: 1.0
+      description: Persists the plan employee assignment details
+      fromTopic: plan-employee-assignment-create-topic
+      isTransaction: true
+      queryMaps:
+        - query: INSERT INTO health.plan_employee_assignment(id, tenant_id, plan_configuration_id, employee_id, role, jurisdiction, additional_details, active, created_by, created_time, last_modified_by, last_modified_time) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?);
+          basePath: $.PlanEmployeeAssignment
+          jsonMaps:
+            - jsonPath: $.PlanEmployeeAssignment.id
+
+            - jsonPath: $.PlanEmployeeAssignment.tenantId
+
+            - jsonPath: $.PlanEmployeeAssignment.planConfigurationId
+
+            - jsonPath: $.PlanEmployeeAssignment.employeeId
+
+            - jsonPath: $.PlanEmployeeAssignment.role
+
+            - jsonPath: $.PlanEmployeeAssignment.jurisdiction
+              type: JSON
+              dbType: JSONB
+
+            - jsonPath: $.PlanEmployeeAssignment.additionalDetails
+              type: JSON
+              dbType: JSONB
+
+            - jsonPath: $.PlanEmployeeAssignment.active
+
+            - jsonPath: $.PlanEmployeeAssignment.auditDetails.createdBy
+
+            - jsonPath: $.PlanEmployeeAssignment.auditDetails.createdTime
+
+            - jsonPath: $.PlanEmployeeAssignment.auditDetails.lastModifiedBy
+
+            - jsonPath: $.PlanEmployeeAssignment.auditDetails.lastModifiedTime
+
+
+    - version: 1.0
+      description: Updates the plan employee assignment details
+      fromTopic: plan-employee-assignment-update-topic
+      isTransaction: true
+      queryMaps:
+        - query: UPDATE health.plan_employee_assignment SET tenant_id = ?, plan_configuration_id = ?, employee_id = ?, role = ?, jurisdiction = ?, additional_details = ?, active = ?, last_modified_by = ?, last_modified_time = ? WHERE id = ?;
+          basePath: $.PlanEmployeeAssignment
+          jsonMaps:
+            - jsonPath: $.PlanEmployeeAssignment.tenantId
+
+            - jsonPath: $.PlanEmployeeAssignment.planConfigurationId
+
+            - jsonPath: $.PlanEmployeeAssignment.employeeId
+
+            - jsonPath: $.PlanEmployeeAssignment.role
+
+            - jsonPath: $.PlanEmployeeAssignment.jurisdiction
```

Comment:
why is jurisdiction JSONB?

Reply:
> why is jurisdiction JSONB?

changed it to string

Reply:
STRING need not be explicitly mentioned, remove it